crates/core/src/ethereum_structs.rs

```diff
@@ -29,13 +32,60 @@ use crate::keccak::KeccakHash;
     Serialize,
     Deserialize,
 )]
+// TODO: move to `namada_ethereum_bridge::event` or
```

Comment:
is there any reason not to resolve this TODO in this PR?

Reply:
some defs in core still depend on this I think. I tried moving this stuff before and it was a bit tricky